src/components/Story.css

```diff
@@ -0,0 +1,15 @@
+.App-story {
+  margin: 0 auto;
+  min-height: 100vh;
+  max-width: 800px;
+  padding: 0 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.App-story p {
+  margin: 1rem auto;
+  font-size: 1.5rem;
+}
```

Comment:
nit--Git likes to have an empty line at the end of each file otherwise it shows a red circle

Reply:
I believe there is a vs code setting for that.  I don't use vscode, so I'm not exactly sure but seems like its: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files